.eslintrc.js

```diff
@@ -9,6 +9,6 @@ module.exports = {
     sourceType: 'module',
   },
```

Comment:
co to za zmiana?

Reply:
Cwiekala Michal, a week ago   (December 16th, 2021 5:21pm) 

prettier + eslint setup